src/sass/lg-fonts.scss

```diff
@@ -1,8 +1,12 @@
 // font icons support
 @font-face {
     font-family: 'lg';
-    src: url("#{$lg-path-fonts}/lg.eot?n1z373");
-    src: url("#{$lg-path-fonts}/lg.eot?#iefixn1z373") format("embedded-opentype"), url("#{$lg-path-fonts}/lg.woff?n1z373") format("woff"), url("#{$lg-path-fonts}/lg.ttf?n1z373") format("truetype"), url("#{$lg-path-fonts}/lg.svg?n1z373#lg") format("svg");
+    src: url("#{$lg-path-fonts}/lg.eot?n1z373"); /* IE9 Compat Modes */
+    src: url("#{$lg-path-fonts}/lg.eot?#iefixn1z373") format("embedded-opentype"), /* IE6-IE8 */
```

Comment:
for cache busting to work fine, shouldn't `n1z373` be part of the query string instead (so using `lg.eot?n1z373#iefix` ?